src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string WorkingDirectory = "r";
```

Comment:
To add one more reason to this. You're adding confusion by overloading the term "Release Directory". Release directory is already a concept today (see variable Agent.ReleaseDirectory and Agent.BuildDirectory).

Combine the "r" folder and numbered folder (e.g. "r1"), and this problem goes away. The term doesn't need to be overloaded; a new term doesn't need to be created.